packages/integration-tests/tests/connector.test.ts

```diff
@@ -100,10 +78,25 @@ test('connector set-up flow', async () => {
    * Change to another SMS/Email connector
    */
   await expect(
-    Promise.all([
-      changeToAnotherConnector(mockSmsConnectorId, mockSmsConnectorConfig, ConnectorType.SMS),
-      changeToAnotherConnector(mockEmailConnectorId, mockEmailConnectorConfig, ConnectorType.Email),
-    ])
+    Promise.all(
```

Comment:
Do we need to wrap up a Promise all here? If anyone fails the test case itself will break as well. Feels like no need to expect not to throw.

Reply:
intend to use promise.all to check it should not fail to set up email/sms/social connector at the same time

Reply:
remove unnecessary `not.to.throw`